mne/viz/epochs.py

```diff
@@ -93,319 +94,376 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
-            evoked=evoked, ts_args=ts_args)
-        figs.append(this_fig)
-
-        # the rest of the code is for aligning ylims for multiple plots
-        if evoked is True and not manual_ylims:
-            evoked_ax = axes_dict["evoked"]
-            this_min, this_max = evoked_ax.get_ylim()
-            curr_min, curr_max = ylims[ch_type]
-            ylims[ch_type] = min(curr_min, this_min), max(curr_max, this_max)
-
-    if evoked is True:  # adjust ylims
-        for group, axes_dict in zip(groups, axes_list):
-            ch_type = group[1]
-            ax = axes_dict["evoked"]
-            this_ymin, this_ymax = these_ylims = ylims[ch_type]
-            ax.set_ylim(these_ylims)
-            yticks = np.array(ax.get_yticks())
-            max_height = yticks[yticks < this_ymax][-1]
+            this_image, style_axes=True, epochs=this_epochs, picks=picks,
+            colorbar=colorbar, vmin=vmin, vmax=vmax, norm=this_norm, cmap=cmap,
+            unit=units[this_ch_type], ax=this_axes_dict, show=False,
+            title=title, combine=combine, combine_given=combine_given,
+            overlay_times=overlay_times, evoked=evoked, ts_args=ts_args)
+        group_by[this_group].update(fig=this_fig)
+
+        # detect ylims across figures
+        if evoked and not manual_ylims:
+            this_bot, this_top = this_axes_dict['evoked'].get_ylim()
+            this_min = min(this_bot, this_top)
+            this_max = max(this_bot, this_top)
+            curr_min, curr_max = auto_ylims[ch_type]
+            auto_ylims[this_ch_type] = [min(curr_min, this_min),
+                                        max(curr_max, this_max)]
+
+    # equalize ylims across figures (does not adjust ticks)
+    if evoked:
+        for this_group_dict in group_by.values():
+            ax = this_group_dict['axes']['evoked']
+            ch_type = this_group_dict['ch_type']
+            this_ymin, this_ymax = ax.get_ylim()
             if not manual_ylims:
-                ax.spines["left"].set_bounds(this_ymin, max_height)
-            if len(vlines) > 0:
-                if overlay_times is not None:
-                    overlay = {overlay_times.mean(), np.median(overlay_times)}
-                else:
-                    overlay = {}
-                for line in vlines:
-                    ax.vlines(line, this_ymin, max_height, colors='k',
-                              linestyles='-' if line in overlay else "--",
-                              linewidth=2. if line in overlay else 1.)
+                args = auto_ylims[ch_type]
+                func = max
+                if 'invert_y' in ts_args:
+                    args = args[::-1]
+                    func = min
+                ax.set_ylim(*args)
+                yticks = np.array(ax.get_yticks())
+                top_tick = func(yticks)
+                # new_ticks = sorted(set([0, top_tick, args[0]]))
+                # ax.set_yticks(new_ticks)
```

Comment:
Forgot this?

Reply:
see comments preceding cell #14 of the gist. Care to weigh in on how to handle this?